server/src/ankaios_server/server_state.rs

```diff
@@ -236,6 +268,15 @@ impl ServerState {
                         ));
                     }
 
+                    self.update_delete_graph(&added_workloads);
+
+                    for workload in deleted_workloads.iter_mut() {
+                        if let Some(delete_conditions) = self.delete_graph.get(&workload.name) {
+                            // [impl->swdd~server-state-adds-delete-conditions-for-deleted-workload~1]
```

Comment:
Same here with the linkage. The whole for loop is there for this functionality.

Reply:
Done.